network-api/networkapi/buyersguide/tests.py

```diff
@@ -72,53 +87,27 @@ def test_product_view_404(self):
         """
         Test that the product view raises an Http404 if the product name doesn't exist
         """
-        request = self.factory.get('/en/privacynotincluded/products/this is not a product')
-        self.assertRaises(Http404, product_view, request, 'this is not a product')
+        response = self.client.get('/en/privacynotincluded/products/this is not a product')
+        self.assertEqual(response.status_code, 404, 'this is not a product')
 
     def test_category_view_404(self):
         """
         Test that the category view raises an Http404 if the category name doesn't exist
         """
-        request = self.factory.get('/en/privacynotincluded/categories/this is not a category')
-        self.assertRaises(Http404, category_view, request, 'this is not a category')
+        response = self.client.get('/en/privacynotincluded/categories/this is not a category')
+        self.assertEqual(response.status_code, 404, 'this is not a category')
 
     def test_category_view(self):
         """
-        Test that the category view returns a 200 for both slug and name URLs
+        Test that the category view returns a 302 for /privacynotincluded/ to /en/privacynotinclded/
+        and return a 404 for a slug that doesn't exist
         """
         response = self.client.get('/privacynotincluded/categories/Smart%20Home/')
-        self.assertEqual(response.status_code, 302, 'The category "Smart Home" should work by name')
```

Comment:
I don't believe we're supporting category names in URLs anymore. We only support slugs. And as far as I can tell we only link to slugged views.

Reply:
that feels right, I don't believe we actually care about supporting spaces etc anymore